src/Access/RowPolicyCache.cpp

```diff
@@ -148,9 +148,12 @@ void RowPolicyCache::ensureAllRowPoliciesRead()
 
     for (const UUID & id : access_control.findAll<RowPolicy>())
     {
-        auto quota = access_control.tryRead<RowPolicy>(id);
-        if (quota)
-            all_policies.emplace(id, PolicyInfo(quota));
+        auto policy = access_control.tryRead<RowPolicy>(id);
+        if (policy)
+        {
+            PolicyInfo policy_info(policy);
+            all_policies.emplace(id, PolicyInfo(policy));
```

Comment:
why do you need to make a copy? Previous variant was moving unnamed object, but otherwise completely functionally equivalent.

Reply:
Thank you, well spotted. It is a kind of typo, policy_info is not used.
The change is just about renaming quota => policy,